src/mqt/qecc/codes/css_code.py

```diff
@@ -161,19 +142,20 @@ def is_self_dual(self) -> bool:
             self.Hx.shape[0] == self.Hz.shape[0] and mod2.rank(self.Hx) == mod2.rank(np.vstack([self.Hx, self.Hz]))
         )
 
-    def stabs_as_pauli_strings(self) -> tuple[list[str] | None, list[str] | None]:
-        """Return the stabilizers as Pauli strings."""
-        x_str = None if self.Hx is None else ["".join(["I" if x == 0 else "X" for x in row]) for row in self.Hx]
-        z_str = None if self.Hz is None else ["".join(["I" if z == 0 else "Z" for z in row]) for row in self.Hz]
-        return x_str, z_str
-
-    def z_logicals_as_pauli_string(self) -> str:
-        """Return the logical Z operator as a Pauli string."""
-        return "".join(["I" if z == 0 else "Z" for z in self.Lx[0]])
+    @staticmethod
+    def _check_valid_check_matrices(Hx: npt.NDArray[np.int8] | None, Hz: npt.NDArray[np.int8] | None) -> None:  # noqa: N803
```

Comment:
should dimensions, orthogonality from ctor be checked here?

Reply:
Sorry, a bit confused by the comment. The check `np.any(Hx @ Hz.T % 2 != 0)` does check for orthogonality. Or am I missing sth?